### PR TITLE
test(lib) add unit tests

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -145,15 +190,128 @@ mod test_utils {
...
+
+        if test_type == TestType::Storage || test_type == TestType::SigningAndStorage {
+            // ---- Stronghold storage ----
+            #[cfg(feature = "stronghold-storage")]
+            test_cases.push(ManagerTestCase::Storage(StorageTestCase {
+                storage_password: None,
+                storage: ManagerStorage::Stronghold,
+            }));
+
+            test_cases.push(ManagerTestCase::Storage(StorageTestCase {
```
Comment (rajivshah3) [inline src/lib.rs]: Is this specific to `stronghold-storage` as well?

Comment (lucasfernog) [reply src/lib.rs]: yeah you're right, thanks for finding that :P

File: src/account/mod.rs [hunk 2]
```diff
@@ -640,35 +640,364 @@ impl Account {
...
+                assert_eq!(
+                    account_handle.read().await.addresses(),
+                    &vec![spent_address, unspent_address1, unspent_address2]
+                );
+            },
+        )
+        .await;
+    }
+
+    // TODO list_addresses tests
```
Comment (Thoralf-M) [inline src/account/mod.rs]: Remove this comment?

File: src/client.rs [hunk 3]
```diff
@@ -254,3 +254,158 @@ pub struct ClientOptions {
...
+    #[test]
+    fn multi_node_constructor() {
+        let nodes = ["https://tangle.iotaqubic.us:14267"];
+        let quorum_size = 5;
+        let quorum_threshold = 0.5;
+        let client = ClientOptionsBuilder::nodes(&nodes)
+            .unwrap()
+            .quorum_size(quorum_size)
+            .quorum_threshold(quorum_threshold)
+            .build()
```
Comment (Thoralf-M) [inline src/client.rs:320]: Should the builder return an error if there is only one node, but quorum_size = 5?

**Comment by Thoralf-M** [comment]
Maybe you should use `https://api.lb-0.testnet.chrysalis2.com` as URL instead of the one with iota1.0 nodes

File: src/account/mod.rs (hunk 2 above)
Comment (bingyanglin) [reply src/account/mod.rs:1002]: Yes please.

File: src/client.rs (hunk 3 above)
Comment (bingyanglin) [reply src/client.rs:320]: I think it's good to have error returning for each function.

File: src/account/mod.rs (hunk 2 above)
Comment (lucasfernog) [reply src/account/mod.rs]: thanks :)

File: src/client.rs (hunk 3 above)
Comment (lucasfernog) [reply src/client.rs:320]: Since iota.rs removed quorum, I'm removing this on another PR too.

**Review by Thoralf-M — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**